pkg/s3wrapper/client.go

```diff
@@ -138,14 +141,19 @@ func (c *S3Client) CreateBucket() error {
 	return nil
 }
 
-func (c *S3Client) UploadStream(ctx context.Context, reader io.Reader, objectName string) error {
+func (c *S3Client) UploadStream(ctx context.Context, reader io.Reader, objectName string, public bool) error {
```

Comment:
usually seeing boolean parameters usage is not clear, maybe you should create a type for it
```
type IsPublic bool
func (c *S3Client) UploadStream(ctx context.Context, reader io.Reader, objectName string, public isPublic) error
```
Then the usage can be something like this `UploadStream(ctx, params.Upfile, fileName, IsPublic(false))`
or 
```
type IsPublic bool
const (
	Public  IsPublic = true
	Private IsPublic = false
)

UploadStream(ctx context.Context, reader io.Reader, objectName string, isPublic IsPublic) error
```

Reply:
Not relevant in new code